README.md

```diff
@@ -20,6 +20,8 @@ Please do not use this code in production, until code goes through more testing
 
 ###Building and Testing
 
+**Note:** Vagrant 1.7.4 and VirtualBox 5.0+ are required to build and test netplugin.
```

Comment:
if we had #119 merged then most likely this would go as an update to the setup document there. Just wondering if we shall pull that doc in first to avoid an outdated doc? But completely up to you.

ping @rschmied as well to see if he still needs help with the merge.